tests/integration/models/reddit/test_wikipage.py

```diff
@@ -24,6 +25,16 @@ def test_edit(self):
         with self.use_cassette():
             page.edit("PRAW updated")
 
+    def test_edit__usernotes(self):
```

Comment:
What do you think about adding a test using the url with the trailing slash that will test for the failure?

Reply:
I didn't include the test that shows that the endpoint with the trailing slash fails, because it wouldn't be a test of PRAW's functionality if the endpoint is modified. It would just be documentation one of many Reddit API bugs/weird behaviors. The API docs themselves list the endpoint without the trailing slash, and there is never a need to add a path after `edit`.

There aren't currently any tests for attempts to edit a normal or special wiki page with oversize payloads. I could add those if you want, or they could be put in a separate PR.

Reply:
That's a good point.

My reasoning was more or less for documentation of said behavior. I think it'd be fine without it.